src/settings/AudioSettings.tsx

```diff
@@ -200,10 +229,37 @@ export default function GeneralSettings(props: ISettingsPanelProps) {
           </IconButton>
         </Tooltip>
       </Box>
+      <Box component="span" sx={{ display: 'flex', alignItems: 'center' }}>
+        <FormControlLabel
+          control={
+            <Switch
+              sx={switchStyle}
+              checked={forceMono}
+              name={forceMonoConfigKey}
+              onChange={(isForceMono) => {
+                setForceMono(Boolean(isForceMono.target.checked));
+                onChange(isForceMono);
+              }}
+            />
+          }
+          label="Force Mono"
+          sx={{
+            color: 'white',
+          }}
+        />
+        <Tooltip title={configSchema.obsForceMono.description}>
+          <IconButton>
+            <InfoIcon sx={{ color: 'white' }} />
+          </IconButton>
+        </Tooltip>
+      </Box>
+      <Box component="h3" sx={headingStyle}>
+        Output Settings
+      </Box>
       <Box component="span" sx={{ display: 'flex', alignItems: 'center' }}>
         <FormControl sx={{ my: 1 }}>
           <InputLabel id="demo-simple-select-label" sx={style}>
-            Out
+            Output Devices
```

Comment:
I took the easy way out here a few weeks back and didn't solve a problem with the border overlap, and instead used some short text. 

I think the below looks a bit sad, so I'd prefer to keep this as "Out" (or better yet, solve the border problem). I spent an hour or so on it embarrassingly and didn't find the solution. 

![image](https://user-images.githubusercontent.com/32389325/214151303-fb2bbe18-c6ae-42c5-9044-6a58c932f8bd.png)

Same comment on the input devices box.

Reply:
I'll take a crack at fixing the border issue. Looks like it could be resolved with css since it displays correctly when the input isn't active.

Reply:
Was able to fix this visual bug - see [this comment](https://github.com/aza547/wow-recorder/pull/305#pullrequestreview-1276461478)

Reply:
Awesome - nice one!